src/smart-components/Channel/components/SuggestedMentionList/index.tsx

```diff
@@ -44,7 +45,8 @@ function SuggestedMentionList(props: SuggestedMentionListProps): JSX.Element {
   const { config, stores } = useSendbirdStateContext();
   const { logger } = config;
   const currentUserId = stores?.sdkStore?.sdk?.currentUser?.userId || '';
-  const { currentGroupChannel } = useChannelContext();
+  // const { currentGroupChannel } = useChannelContext();
+  const currentGroupChannel = useChannelContext?.()?.currentGroupChannel || useThreadContext?.()?.currentChannel;
```

Comment:
not a good pattern, might cause issue in the future

Reply:
!important

Reply:
I changed it, could you check it again